src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7136,4 +7136,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_UnsupportedCompilerFeature" xml:space="preserve">
     <value>'{0}' requires compiler feature '{1}', which is not supported by this version of the C# compiler.</value>
   </data>
+  <data name="WRN_ObsoleteMembersShouldNotBeRequired" xml:space="preserve">
+    <value>Members attributed with 'ObsoleteAttribute' should not be required unless the containing type is obsolete or all constructors are obsolete.</value>
```

Comment:
Consider including the symbol for the member which had both required and obsolete on it here, i.e. as argument `{0}`.